src/components/todosFilter/TodosFilter.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../../styles/filter.css';
+import { FilterStatus } from '../../types/FilterStatus';
+
+export const TodosFilter: React.FC = () => {
+  return (
+    <>
+      <ul className="filters" data-cy="todosFilter">
+        <li>
+          <NavLink
+            to="/"
+            className={({ isActive }) => (isActive ? 'selected' : '')}
+          >
```

Comment:
May be move it to separate callback? We use it several times

Reply:
agree